### DISCUSSION TITLE
Help vs docs Folder naming convention

### DISCUSSION THREAD

**Discussion by trwbox** [ainfosec/FISSURE#40] [General]
I've noticed that the Help folder contains what I would describe as the documentation for FISSURE, would it make sense to change it to the more common `docs/` naming scheme as seen in [QMK](https://github.com/qmk/qmk_firmware/tree/master/docs), [NGINX](https://github.com/nginx/nginx/tree/master/docs), [HackRF](https://github.com/greatscottgadgets/hackrf/tree/master/docs), etc?

It could also be a good place to put the contents of the current `Gallery/` folder used by the README and any other misc images, links, pdfs, lesson, etc. To help declutter where all the non-code files are stored.

**Comment by cpoore1** [comment]
Sure, I can move them. I'll be spinning up a more in-depth Read the Docs user manual soon which will create the docs folder anyway. I'll have to see later on if it makes sense to merge some of those pages into the user manual.
